build failed with cmake3.15

When compiling libevent on a CentOS 7 server, the compilation fails because the CMake version on the server is only 3.17.5, and the CheckLinkerFlag module used in cmake/AddLinkerFlags.cmake is a new addition in CMake 3.18. I noticed that the minimum required CMake version specified in the project's CMake configuration is 3.10. Therefore, I think this might be a bug.

```
/usr/bin/cmake3 -DCMAKE_BUILD_TYPE=Debug -G "CodeBlocks - Unix Makefiles" -S /home/felix/tmp/tmp.58peda5hgT/libevent -B /home/felix/tmp/tmp.58peda5hgT/libevent/cmake-build-debug-docker-cpp
-- The C compiler identification is GNU 4.8.5
-- Check for working C compiler: /usr/bin/cc
-- Check for working C compiler: /usr/bin/cc - works
-- Detecting C compiler ABI info
-- Detecting C compiler ABI info - done
-- Detecting C compile features
-- Detecting C compile features - done
CMake Error at cmake/AddLinkerFlags.cmake:1 (include):
  include could not find load file:

    CheckLinkerFlag
Call Stack (most recent call first):
  CMakeLists.txt:66 (include)


-- Found Git: /usr/bin/git (found version "1.8.3.1") 
CMake Error at cmake/AddLinkerFlags.cmake:7 (check_linker_flag):
  Unknown CMake command "check_linker_flag".
Call Stack (most recent call first):
  CMakeLists.txt:208 (add_linker_flags)


-- Configuring incomplete, errors occurred!
See also "/home/felix/tmp/tmp.58peda5hgT/libevent/cmake-build-debug-docker-cpp/CMakeFiles/CMakeOutput.log".
```

--- a/cmake/AddLinkerFlags.cmake
+++ b/cmake/AddLinkerFlags.cmake
@@ -4,6 +4,7 @@ macro(add_linker_flags)
 	foreach(flag ${ARGN})
 		string(REGEX REPLACE "[-.+/:= ]" "_" _flag_esc "${flag}")
 
+# Let's make Centos7 users (cmake 3.17) happy
 if (NOT CMAKE_VERSION VERSION_LESS 3.18)
 		check_linker_flag(C "${flag}" check_c_linker_flag_${_flag_esc})
 endif()

